gn3/computations/datasets.py

```diff
@@ -249,6 +248,11 @@ def divide_into_chunks(the_list, number_chunks):
     return chunks
 
 
+def escape(string_):
+    """function escape sql value"""
+    return escape_string(string_).decode('utf8')
+
+
```

Comment:
When you use the `%s` syntax provided my mysqldb, it does the escaping for you. You'd have to confirm that though...